src/test/java/com/faforever/api/voting/VotingServiceTest.java

```diff
@@ -272,42 +272,4 @@ public void notSaveVoteOnTooManyAnswers() {
     }
     verify(voteRepository, never()).save(vote);
   }
-
-  @Test
-  public void notSaveOnVoteTwiceInOneOption() {
```

Comment:
(Why) is this test obsolete?

Reply:
With to new equal method kubko did it gets obsolete

Reply:
Ok. Accepted :)